people.rkt

```diff
@@ -71,7 +71,7 @@
                   #:title "Professor"
                   #:website "http://ccs.neu.edu/home/will"
                   #:e-mail "will@ccs.neu.edu"
-                  #:history @list[@a[href: "http://www.cesura17.net/~will/Professional/"]{www.cesura17.net/~will/Professional/}]
+                  #:history @list[]
```

Comment:
Oh, I see. I mistook this to be the URL under their name, not the history list to the side. This is fine.